eth_test_parser/src/arg_parsing.rs

```diff
@@ -1,5 +1,9 @@
 use clap::Parser;
 
 #[derive(Debug, Parser)]
-#[clap(author, version, about)]
-pub(crate) struct ProgArgs {}
+#[command(author, version, about)]
+pub(crate) struct ProgArgs {
+    // #[arg(short, long, default_value_t = false)]
```

Comment:
Nit. Probably forgot to take this out.

Also a bit optional, but it might be worth adding a doc comment to the arg since `Clap` uses this as a description.